modules/gdpr_tasks/src/Traversal/RightToAccessDisplayTraversal.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Drupal\gdpr_tasks\Traversal;
+
+use Drupal\Core\Entity\FieldableEntityInterface;
+use Drupal\gdpr_fields\Entity\GdprFieldConfigEntity;
+use Drupal\gdpr_fields\EntityTraversal;
+
+/**
+ * Entity traversal for the Right to Access preview display.
+ *
+ * @package Drupal\gdpr_tasks
+ */
+class RightToAccessDisplayTraversal extends EntityTraversal {
```

Comment:
Do we need this if we are building the CSVs first? Can we just render the CSVs like we do in D7? This could be quite intensive at run time on a large system...

Reply:
Will leave it to @yanniboi to answer this one.

Reply:
Lets fix this as part of the CSVs update.